### PR TITLE
lua-lsqlite3: add build VARIANT for Lua5.3

### PR DESCRIPTION
**Pull request by zhaojh329**
Maintainer: @oskarirauta
Compile tested: x86, x86, master
Run tested: x86, x86, master, tests done

Description:

### CODE REVIEW

**Comment by oskar456** [comment]
Maintainer: @oskarirauta

**Comment by zhaojh329** [comment]
@1715173329
Could you merge this PR?

**Review by 1715173329 — APPROVED** [review]

**Comment by 1715173329** [comment]
I'll merge this tonight if there's no objection.

File: lang/lua-lsqlite3/Makefile
Comment (1715173329) [inline lang/lua-lsqlite3/Makefile:77]: `/usr/local`? this path seems unused on openwrt

Comment (zhaojh329) [reply lang/lua-lsqlite3/Makefile:77]: The default prefix is `/usr/local` in Lua5.3.

Comment (zhaojh329) [reply lang/lua-lsqlite3/Makefile:77]: image

Comment (1715173329) [reply lang/lua-lsqlite3/Makefile:77]: It might be better to patch the [lua5.3](https://github.com/openwrt/openwrt/tree/main/package/utils/lua5.3) package I think, just like what we did in [lua5.1](https://github.com/openwrt/openwrt/blob/main/package/utils/lua/patches/200-lua-path.patch).

Comment (zhaojh329) [reply lang/lua-lsqlite3/Makefile:77]: There are some packages already use it. Such as luasocket-lua-53.

Comment (1715173329) [reply lang/lua-lsqlite3/Makefile:77]: Hmmm okay.

**Pull request merged**
